### ISSUE TITLE
Broken Help Info icon next to negative grant values on Grant page

### ISSUE THREAD

**Issue by R2ZER0** [ThreeSixtyGiving/grantnav#1114]
**Describe the bug**
The help info icon does not load due to broken image URL.

**To Reproduce**
1. Go to https://grantnav.threesixtygiving.org/grant/360G-cabinetoffice-GA-094576 (or any other Grant page with a negative value)
3. Scroll down to the Grant Details table, next to the Amount Awarded value
4. The icon https://grantnav.threesixtygiving.org/static/images/icon-help-50.png does not load.

**Expected behavior**
The icon should load and display.

**Screenshots**
<img src="https://i.ibb.co/jDJxS6s/image.png"></img>

**Comment by jarofgreen** [comment]
Looks like this has already been sorted by something else?

**Comment by michaelwood** [comment]
Yep this is now fixed. The link needs updating though so that's now in https://github.com/ThreeSixtyGiving/grantnav/pull/1201
